### ISSUE TITLE
[Manga Library Z] Connector not working correctly?

### ISSUE THREAD

**Issue by l-l-l-l-l-l** [manga-download/hakuneko#3998]
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
[Yes]/No

**Is the website of the connector working properly / are you able to see the manga within your browser**
[Yes]/No

**Describe the bug**
This is weird, but there are some pages that don't show properly, causing the download to fail upon reaching them. No matter how many times I repeat the download or reset Hakuneko, it's always the exact same pages (different for every chapter). They work correctly on my browser.

**To Reproduce**
1. Manga Library Z
2. Any (although I managed once to download just one chapter from one after many retriesw, but no luck with anything else).
3. Any.
4. I checked one of them with the advanced console, loading directly one of the broken pages (which is shown as perpetually loading). This is the log I get from the advanced console:
[vw.mangaz.com-1632579511750.log](https://github.com/manga-download/hakuneko/files/7229876/vw.mangaz.com-1632579511750.log) 

**Screenshots**
![image](https://user-images.githubusercontent.com/36646277/134774778-6a3f6904-fc61-4783-8c23-62e3250b7e34.png)

**HakuNeko :**
 - 6.17@c86d92

**Comment by 27136773** [comment]
Please provide sample manga/chapter

**Comment by l-l-l-l-l-l** [comment]
The example from that specific log would be volume 5 of this one series: https://www.mangaz.com/series/detail/136501
but it happens with anything else I've tried (different pages in each case, but the same every time for any given volume). In this particular case it's always pages 28, 31, 35, 145, 174 and over half of page 33, according to the preview.
I tried it on my browser, but those load correctly.

**Comment by l-l-l-l-l-l** [comment]
Could it be possible to have Hakuneko just asking for the missing images in case of a botched download instead of scrapping everything and trying again, in case that that's what's wrong (at least the warning it gives indicates that that's what it does)?

**Comment by l-l-l-l-l-l** [comment]
I seem to have found the issue.
Just to check if it had something to do with my connection, I downloaded a portable copy and imported the bookmarks. I first tried to download one manga from the site still from my PC and got the same "failed to fetch" error. When I went to a different computer, I found that other mangas either downloaded straight out or gave a "failed to load after 60 seconds" error that was fixed just by reattempting a couple of times, even those that had failed on my computer. Except one. The one I'd tried with my computer for this instance? That one keeps giving me the "failed to fetch" error again and again, always at the same point.
It seems that, for some reason, the connector prevents hakuneko from downloading anything that gave a "failed to fetch" error before.
